MolecularNodes/md.py

```diff
@@ -250,6 +335,26 @@ def att_is_peptide():
                 warnings.warn("Unable to add custom selection: {}".format(sel.name))
 
     coll_frames = coll.frames(name)
+
+
+
+
+    def bool_selection_2(selection):
```

Comment:
This is very similar to `bool_selection`. Why the separate function? I'd advise formatting your data so that the same `bool_selection` function can be used.